data/attributes.yml

```diff
@@ -32352,6 +32352,27 @@ base_attributes:
   - broadhead_design__fixed_blade
   - broadhead_design__mechanical
   - broadhead_design__other
+- id: 4678
+  name: Scooter brake type
+  description: Specifies the mechanism and design of scooter brakes, such as disc or drum
+  friendly_id: scooter_brake_type
+  handle: scooter-brake-type
+  values:
+  - scooter_brake_type__combination_brakes
+  - scooter_brake_type__disc_brakes
+  - scooter_brake_type__drum_brakes
+  - scooter_brake_type__foot_brakes
+  - scooter_brake_type__regenerative_brakes
+  - scooter_brake_type__other
+- id: 4679
+  name: Electric scooter motors type
+  description: Specifies the type and configuration of scooter motors, such as brushed DC or brushless dc
+  friendly_id: electric_scooter_motors_type
+  handle: electric-scooter-motors-type
+  values:
+  - electric_scooter_motors_type__brushed_dc_motor
+  - electric_scooter_motors_type__brushless_dc_motor
+  - electric_scooter_motors_type__other
```

Comment:
Not sure about the importance of this one as a standard attribute, I can't find enough examples?